### PR TITLE
Add an OEP-2 compliant openedx.yaml file

### PR DESCRIPTION
**Pull request by cpennington**
This adds an `openedx.yaml` file, as described by OEP-2:
http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html

The data in this file was transformed from the contents of
edx/repo-tools-data:repos.yaml

### CODE REVIEW

**Comment by cpennington** [comment]
@edx/platform-team: I've tagged you as the owner of this repo. Ok to merge?

**Comment by doctoryes** [comment]
So why platform and not ecom or mobile, both who've done more work on this repo? Maybe it could transition to platform over time - but it seems to belong to one of those teams at the moment.

**Comment by cpennington** [comment]
fair. @edx/ecommerce: Would you guys be ok w/ taking this for now?

File: openedx.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: oau
+oeps: {}
+owner: edx/platform-team
```
Comment (clintonb) [inline openedx.yaml]: edx/ecommerce

**Comment by clintonb** [comment]
👍

**Pull request merged**